src/atomic.py

```diff
@@ -79,6 +79,10 @@ def look_at_point(x_node, y_node, z_node):
 
 def look_at_face_point(x, y, z):
 	evl.look_at_point(x, y, z)
+	#evl.gaze_at_point(x, y, z)
```

Comment:
OK.  Well, "look at" by definition means "turn the whole head". The spec says so, clearly and unambiguously, and so does the code, in multiple places. This has not changed in many years, unless the guys dinking with the blender model  broke something.  You can run the demos. Have you tried running the demos? They are here:  

https://github.com/hansonrobotics/blender_api_msgs/blob/master/cookbook.md

docs are here:
https://github.com/hansonrobotics/blender_api/blob/master/docs/API_v1.md

Can you verify that both of these got copied into the hansonrobotics/HEAD repo?  They should be there -- you should not have to guess how this stuff works. Its supposed to all be documented.
---
Anyway, please be aware that this/these python calls are also used by the chatbot, so that if you tell the chatbot "turn to the left" then her whole head should turn, and if you say "look to the left" then only her eyes should move.  So you really should not be dinking with these calls, as doing so will likely screw up the chatbot.